elk/utils_evaluation/utils_evaluation.py

```diff
@@ -32,7 +31,7 @@ def get_filtered_filenames(
     :param confusion: the confusion
     :param place: the place where the data is from
     """
-    files = map(lambda path: path.name, directory.iterdir())
+    files = map(lambda path: str(path.relative_to(directory)), directory.glob("**/"))
```

Comment:
What was the issue with the previous line of code?

Reply:
The new code searches for files _recursively_ while the previous code only looked at the top level. I think.

Reply:
That's it, a path like `microsoft/deberta` becomes a two-level directory. We could replace slashes with some other character but imo that wouldn't be worth keeping it up everywhere.